sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/ReceivedDeliveryOutcome.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+/**
+ * Represents a partial message that was received.
+ *
+ * @see DeliveryState
+ * @see <a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-received">Received
+ *     outcome</a>
+ */
+public class ReceivedDeliveryOutcome extends DeliveryOutcome {
```

Comment:
Is there another outcome that has sub-classed this?

Reply:
No. This can be final too. Good catch.